<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UILogin</class>
 <widget class="QWidget" name="UILogin">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>216</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PChat 登录</string>
  </property>
  <widget class="QLineEdit" name="txtUsername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPasswd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_username">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>54</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;用户名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_passwd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>102</y>
     <width>49</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnForget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
